src/compiler/compiler.lisp

```diff
@@ -1427,8 +1441,16 @@
 (define-builtin objectp (x)
   (convert-to-bool `(=== (typeof ,x) "object")))
 
+;;; @vlad-km
 (define-builtin js-undefined-p (x)
-  (convert-to-bool `(=== ,x undefined)))
+  (convert-to-bool `(== (typeof ,x) "undefined")))
+
+;;; return `null`
+(define-builtin %get-js-null% () ` (call-internal |makJSnull|))
+;;; return `undefined`
+(define-builtin %get-js-undef% () `(call-internal |makJSundef|))
+;;; return object with props `null/undef` and access methods
+(define-builtin %get-js-bvo% ()   `(call-internal |badJSvalues|))
```

Comment:
Maybe it's obvious but, what does `bvo` stand for?
bad values object?


any reason why we need bvo instead of the individual null and undef you created?

Reply:
It's an artifact from an old implementation. will be removed